### PR TITLE
feat(app): databricks notebook support

### LINKED ISSUE #953: Support Databricks Notebooks
Wondering if Phoenix support for Databricks is on the roadmap, after hitting a few issues here

### PR DESCRIPTION
**Pull request by mikeldking**
resolves #953 

Screenshot 2024-01-12 at 11 44 36 PM

### CODE REVIEW

**Comment by mikeldking** [comment]
Figured out the driver proxy URL but getting a 502 bad gateway. Process session fails entirely.

File: src/phoenix/config.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def get_running_pid() -> Optional[int]:
 # Server config
 SERVER_DIR = PHOENIX_DIR / "server"
 # The host the server will run on after launch_app is called
-HOST = "127.0.0.1"
+HOST = "0.0.0.0"
```
Comment (mikeldking) [inline src/phoenix/config.py:51]: confirmed this works in all environments

File: src/phoenix/server/templates/index.html [hunk 2]
```diff
@@ -21,6 +21,10 @@
   <body>
     <div id="root"></div>
     <script>(function() {
+        // Request Info:
+        // URL: {{request.url}}
+        // Base URL: {{request.base_url}}
+        // Root Path: {{root_path}}
```
Comment (mikeldking) [inline src/phoenix/server/templates/index.html:27]: helpful for debugging. Gonna leave in

File: src/phoenix/services.py [hunk 3]
```diff
@@ -119,6 +120,7 @@ def __init__(
         self.export_path = export_path
         self.host = host
         self.port = port
+        self.root_path = root_path  # TODO(mikeldking): Add support for root_path
```
Comment (mikeldking) [inline src/phoenix/services.py:123]: Process server seems to be largely broken again. Punting

**Review by anticorrelator — APPROVED** [review]
lgtm!

**Pull request merged**